dev/release/post-03-website.sh

```diff
@@ -55,14 +57,37 @@ rough_n_development_months=$((
 git_tag=apache-arrow-${version}
 git_range=apache-arrow-${previous_version}..${git_tag}
 
-committers_command_line="git shortlog -csn ${git_range}"
-contributors_command_line="git shortlog -sn ${git_range}"
-
-committers=$(${committers_command_line})
-contributors=$(${contributors_command_line})
+directories=("${ARROW_DIR}" "${ARROW_RS_DIR}")
+git_ranges=(apache-arrow-${previous_version}..${git_tag} ${previous_version}..${version})
+
+committers=$(
+  for (( i=0; i<${#directories[@]}; i++ ));
+  do
+    cd ${directories[$i]}
+    git shortlog -csn ${git_ranges[$i]}
+  done | 
+  awk '{
+    name_email="";
+    for (i=2; i<=NF; i++) { name_email=name_email " " $i}; 
+    count_by_user[name_email]+=$1} END {for (name_email in count_by_user) print count_by_user[name_email], name_email}' |
+  sort -rn 
+)
+
+contributors=$(
+  for (( i=0; i<${#directories[@]}; i++ ));
+  do
+    cd ${directories[$i]}
+    git shortlog -sn ${git_ranges[$i]}
+  done | 
+  awk '{
+    name_email="";
+    for (i=2; i<=NF; i++) { name_email=name_email " " $i}; 
+    count_by_user[name_email]+=$1} END {for (name_email in count_by_user) print count_by_user[name_email], name_email}' |
```

Comment:
Would it make sense to pull this `awk` command out into a variable so we only have to specify it once?

Reply:
Good point, I will do this!

Reply:
Done